Source/BugsnagClient.m

```diff
@@ -217,6 +252,23 @@ @interface BugsnagConfiguration ()
 
 @implementation BugsnagClient
 
+#if TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE
+/**
+ * Storage for the device orientation.  It is "last" whenever an orientation change is received
+ */
+static NSString *_lastOrientation = nil;
```

Comment:
Why does this need to be static? Is it possible to just assign to `self.lastOrientation` declared in the class extension instead?

Reply:
Previous discussion agreed that this should be a class property.  My understanding is that class properties need static storage.  Am I misunderstanding something?

Reply:
Apologies - my terminology might be a little off as it's a while since I've used Objective-C in anger. The intent behind my review comment is that this would be defined in a [class extension](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW3), similar to this:

```
// within BugsnagClient.m

@interface BugsnagClient()
@property NSString *lastOrientation;
@end
```

This would avoid exposing the property publicly. Adding new static variables will make it harder to instantiate multiple objects of `BugsnagClient` - which is something we may want to do eventually. As far as I'm aware using a class extension wouldn't require static storage?

Reply:
Discussed offline, moved from class to instance-level storage.